features/sign_up.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Signup to the webmail
+As a visitor
```

Comment:
use proper indentation in this file

Reply:
Changes done accordingly. Thank you for your valid input.